cst438-project-airline/src/main/java/cst438/domain/ReservationRepository.java

```diff
@@ -0,0 +1,7 @@
+package cst438.domain;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ReservationRepository extends JpaRepository<Reservation, Long> {
+
```

Comment:
We'll need a query to select all the reservations a user name grouping it by date.
Maybe we can add an aggregate query to get the total of a cost, but I think we can do that on the service end.